auto_submit/lib/validations/ci_successful.dart

```diff
@@ -201,40 +184,10 @@ class CiSuccessful extends Validation {
         // checkrun has failed.
         log.info('${slug.name}/$prNumber: CheckRun $name failed.');
         failures.add(FailureDetail(name!, checkRun.detailsUrl as String));
-      } else if (checkRun.status == github.CheckRunStatus.queued) {
-        if (isStale(checkRun.startedAt) && supportStale(author, slug)) {
-          staleCheckRuns.add(checkRun);
-        }
       }
       allSuccess = false;
     }
-    if (staleCheckRuns.isNotEmpty) {
-      log.warning(
-        'Pull request https://github.com/${slug.fullName}/pull/$prNumber from ${slug.name} repo auto roller has been running over ${Config.kGitHubCheckStaleThreshold} hours due to: ${staleCheckRuns.map((e) => e.name).toList()}',
-      );
-    }
 
     return allSuccess;
   }
-
-  // Treat any GitHub check run as stale if created over [Config.kGitHubCheckStaleThreshold] hours ago.
-  bool isStale(DateTime dateTime) {
```

Comment:
I am not sure about the removal of this here. @keyonghan what do you think?

Reply:
If I recall correctly we are trying to flag PRs that are over 2 hours old so I think this code needs to stay if you are only removing stuff related to flutter-gold.

Reply:
This was my bad, I had branched from a stale PR. Reverted.